assets/js/modules/adsense/components/dashboard/AdSenseDashboardWidget.js

```diff
@@ -119,6 +119,24 @@ class AdSenseDashboardWidget extends Component {
 		// Hide AdSense data display when we don't have data.
 		const wrapperClass = ( loading || ! receivingData || zeroData ) ? 'googlesitekit-nodata' : '';
 
+		let moduleStatus;
+		let moduleStatusText;
+		if ( ! error && modulesData.adsense.setupComplete ) {
+			moduleStatus = 'connected';
+			moduleStatusText = sprintf(
+				/* translators: %s: module name. */
+				__( '%s is connected', 'google-site-kit' ),
+				_x( 'AdSense', 'Service name', 'google-site-kit' )
```

Comment:
Should we maybe use `module.name` here instead of translating it again?

Reply:
That's a good point, however I'd say let's change that separately. There are a bunch of those references, and I think we could fix those throughout. IMO this makes for a good distinct follow-up issue only focused on that.